scripts/validate_data.py

```diff
@@ -54,7 +55,22 @@ def validate_data(data_directory):
     # Calculate actual hash for given filename.
     # If hash for filename is not the same as the one in the file, raise
     # ValueError
-    raise NotImplementedError('This is just a template -- you are expected to code this.')
+    hash_file_path = os.path.join(data_directory, "group-00/hash_list.txt")
```

Comment:
I believe there is a mismatch between the function description and what is done as `data_hashes.txt` is expected to be found in `data_directory` (which is supposed to be `data` from the call `validate_data.py data`) but is in `data/group-00` instead. That's probably not intended for us to consider this but it's just to bring the potential issue when having multiple datasets.

Reply:
yes good to keep track of this, at some point we'll maybe get the other datasets